includes/admin/class-wc-payments-admin.php

```diff
@@ -181,11 +181,11 @@ public function register_payments_scripts() {
 			'WCPAY_DASH_APP',
 			'wcpaySettings',
 			[
-				'connectUrl'         => WC_Payments_Account::get_connect_url(),
```

Comment:
`wcpaySettings.connectUrl` is used [here](https://github.com/Automattic/woocommerce-payments/blob/c7f9d58e2d20fc5fb10d3fa5d205f698bcf3b8c6/client/connect-account-page/index.js#L75).

Reply:
Thanks – no idea how that happened! Fixed in https://github.com/Automattic/woocommerce-payments/pull/925/commits/cbe11d15b29051d0ab8187f174c1841da0daa89c